app/index.js

```diff
@@ -16,7 +16,7 @@ updateNotifier();
 
 // Grab a list of all CF components, we'll use it later.
 var components = request({
-  uri: 'https://api.github.com/orgs/cfpb/repos?per_page=100',
+  uri: 'https://api.github.com/search/repositories?q=%22cf-%22+NOT+cfpb+NOT+cfgov+NOT+deprecated+user:cfpb+language:css',
```

Comment:
I only set the language to keep with [this chunk](https://github.com/cfpb/generator-cf/blob/master/app/lib/filter-components.js#L14-L15). I figured each of these would have docs so each should also have css files. I could go with `js` since each should have Bower and NPM manifests.